[2025-05-14T14:55:00.960+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T14:50:00+00:00 [queued]>
[2025-05-14T14:55:00.971+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T14:50:00+00:00 [queued]>
[2025-05-14T14:55:00.972+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T14:55:00.975+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-05-14T14:55:00.975+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T14:55:00.990+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): Extract_Data> on 2025-05-14 14:50:00+00:00
[2025-05-14T14:55:00.996+0000] {standard_task_runner.py:52} INFO - Started process 21463 to run task
[2025-05-14T14:55:00.999+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Sales_Performance_ETL', 'Extract_Data', 'scheduled__2025-05-14T14:50:00+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Hisham_Audiputra_Suharto_DAG.py', '--cfg-path', '/tmp/tmptag65tj7', '--error-file', '/tmp/tmp502qy22k']
[2025-05-14T14:55:01.002+0000] {standard_task_runner.py:80} INFO - Job 93: Subtask Extract_Data
[2025-05-14T14:55:01.074+0000] {task_command.py:371} INFO - Running <TaskInstance: Sales_Performance_ETL.Extract_Data scheduled__2025-05-14T14:50:00+00:00 [running]> on host 2ca7303636ab
[2025-05-14T14:55:01.173+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Hisham
AIRFLOW_CTX_DAG_ID=Sales_Performance_ETL
AIRFLOW_CTX_TASK_ID=Extract_Data
AIRFLOW_CTX_EXECUTION_DATE=2025-05-14T14:50:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-14T14:50:00+00:00
[2025-05-14T14:55:01.174+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-05-14T14:55:01.175+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/scripts/Extract_Data.py']
[2025-05-14T14:55:01.185+0000] {subprocess.py:85} INFO - Output:
[2025-05-14T14:55:01.976+0000] {subprocess.py:92} INFO - Warning: Looks like you're using an outdated `kagglehub` version, please consider updating (latest version: 0.3.12)
[2025-05-14T14:55:01.992+0000] {subprocess.py:92} INFO - Path to dataset files: /home/***/.cache/kagglehub/datasets/zahidmughal2343/supplement-sales-data/versions/1
[2025-05-14T14:55:01.995+0000] {subprocess.py:92} INFO - Path to dataset files: /home/***/.cache/kagglehub/datasets/zahidmughal2343/supplement-sales-data/versions/1
[2025-05-14T14:55:02.009+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-05-14T14:55:02.045+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Sales_Performance_ETL, task_id=Extract_Data, execution_date=20250514T145000, start_date=20250514T145500, end_date=20250514T145502
[2025-05-14T14:55:02.097+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-14T14:55:02.157+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
